app/views/finders/_facet_tags.html.erb

```diff
@@ -0,0 +1,26 @@
+<% if local_assigns[:applied_filters] %>
+  <div class="facet-tags" data-module="track-click">
+    <% local_assigns[:applied_filters].each do |applied_filter| %>
+      <div class="facet-tags__group">
+        <% applied_filter.each do |filter| %>
+          <div class="facet-tags__wrapper">
+            <p class="facet-tags__preposition"><%= filter[:preposition] %></p>
+            <div class="facet-tag">
+              <p class="facet-tag__text"><%= filter[:text] %></p>
+              <button type="button"
+                class="facet-tag__remove"
+                aria-label="Remove filter <%= filter[:text] %>"
+                data-module="remove-filter-link"
+                data-track-label="<%= filter[:data_track_label] %>"
+                data-facet="<%= filter[:data_facet] %>"
+                data-value="<%= filter[:data_value] %>"
+                data-name="<%= filter[:data_name] %>">&#x2715;</button>
+            </div>
+          </div>
+        <% end %>
+      </div>
+    <% end %>
+  </div>
+<% end %>
+
+<%= local_assigns[:screen_reader_filter_description].html_safe %>
```

Comment:
There's a fair amount of `html_safe` being used. We should do an audit of these calls to ensure that the user input is escaped upstream of this.

Reply:
I've removed this instance of `html_safe`.